src/index.js

```diff
@@ -7,8 +7,8 @@ if (!('import' in document.createElement('link'))) {
   require('../lib/vendor/HTMLImports');
 }
 
-require('../style/aframe-core.css');
-require('../style/rStats.css');
+require('./style/aframe-core.css');
```

Comment:
you can remove [`style` from the budo script watch list](https://github.com/aframevr/aframe/blob/dev/scripts/budo.js#L19)